<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADC</class>
 <widget class="QWidget" name="ADC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="sHelp">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>844</width>
        <height>178</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Usage: adc &lt;adc1/tempsensor/vrefint/vbat&gt; [period (nb ms)] [samples (nb sample)/continuous]
   adc1           ADC1 (PA1)
   tempsensor     Temperature sensor
   vrefint        Internal reference voltage
   vbat           VBAT voltage
   period         Delay between samples (msec)
   samples        Number of samples
   continuous     Read continuously
</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QChartView" name="sADC_Graph"/>
   </item>
   <item>
    <widget class="QWidget" name="control" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="cPlot">
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="cADC1">
           <property name="text">
            <string>PA1</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cTemp">
           <property name="text">
            <string>Temp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cVRef">
           <property name="text">
            <string>V Ref</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cVBat">
           <property name="text">
            <string>V Bat</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Period, ms</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sPeriod">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Samples</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sSamples">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkUnlim">
        <property name="text">
         <string>cont</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>129</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bADC">
        <property name="text">
         <string>ADC</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bADC</sender>
   <signal>clicked()</signal>
   <receiver>ADC</receiver>
   <slot>cmd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>818</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cmd()</slot>
 </slots>
</ui>
